client/actions/fish.js

```diff
@@ -13,6 +13,13 @@ export function getFishSuccess (fish) {
   }
 }
 
+export function getFishDetailSuccess (fishDetail) {
+  return {
+    type: 'GET_FISH_DETAIL',
```

Comment:
It's probably not a huge deal, but in the future consider naming the `type` using the same pattern as the action creator function name. So:
```js
export function fishDetailsSuccess (details) {
   return {
    type: 'FISH_DETAILS_SUCCESS',
    details
  }
}
```
Consistency really helps when you're trying to debug things :smile: